core/embed/projects/firmware/main.c

```diff
@@ -76,6 +76,7 @@ int main(uint32_t cmd, void *arg) {
   // Interpreter init
   printf("CORE: Starting interpreter\n");
   mp_init();
+
```

Comment:
maybe omit this noise?

Reply:
Oops, my bad... I removed the code from firmware build, instead of disabling it for non-debug builds.